CONTRIBUTORS.md

```diff
@@ -58,3 +58,4 @@ From oldest to newest contributor, we would like to thank:
 - [Nicole Mazzuca](https://github.com/ubsan)
 - [Jameson Nash](https://github.com/vtjnash)
 - [Marc Tiehuis](https://github.com/tiehuis)
+- [Niek J. Bouman](https://github.com/niekbouman)
```

Comment:
Cool! The `CONTRIBUTORS.md` is actually updated by `etc/scripts/contributorer.py` but I'm delighted to have yo uhere (we only run it on demand so your name would turn up when we next remember)